src/pykeen/datasets/cli.py

```diff
@@ -255,6 +256,9 @@ def expected_metrics(dataset: str, max_triples: Optional[int]):
             df.to_csv(output_path, sep="\t", index=False)
 
             # expected metrics
+            ks = (1, 3, 5, 10) + tuple(
+                10 ** i for i in range(2, int(math.ceil(math.log(dataset_instance.num_entities))))
```

Comment:
will the final k be bigger than num entities?

Reply:
The upper bound of range is exclusive